app.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const { getTopics } = require('./controllers/topics-controllers');
+const app = express()
+
+app.use(express.json())
+
+app.get('/api/topics', getTopics);
+
+app.use((err, req, res, next) => {
+    if (err) res.status(500).send({ err })
+})
```

Comment:
Same as above, you currently don't have any custom errors to catch in this ticket - so this custom error handler should be removed